packages/frontend/pages/items/[slug]/index.tsx

```diff
@@ -4,16 +4,19 @@ import { QueryWrapper } from "@wdp/lib/api/components";
 import { useRouteSlug } from "@wdp/lib/routes";
 import CommunityChildLayout from "components/composed/community/CommunityChildLayout";
 import { SlugItemQuery as Query } from "@/relay/SlugItemQuery.graphql";
-import EntityLayoutFactory from "components/factories/EntityLayoutFactory";
+import EntityContentLayoutFactory from "components/factories/EntityContentLayoutFactory";
```

Comment:
This can probably get renamed to EntityContentIndexFactory or something like that. Not sure if "Content" is clear in this case.